docs/public/static/sponsors/aptugo-dark.svg

```diff
@@ -0,0 +1,10 @@
+<path d="M109.59 17.4919C109.236 16.8066 108.734 16.2736 108.095 15.9061C107.46 15.5408 106.72 15.3559 105.897 15.3559C105.073 15.3559 104.333 15.5408 103.698 15.9061C103.06 16.2724 102.558 16.8056 102.204 17.4908C101.855 18.1658 101.679 18.9846 101.679 19.9261C101.679 20.8664 101.852 21.6843 102.195 22.3581C102.543 23.0443 103.041 23.5786 103.674 23.9461C104.303 24.3103 105.051 24.495 105.897 24.495C106.721 24.495 107.46 24.3113 108.095 23.9461C108.733 23.5797 109.236 23.0455 109.59 22.3604C109.939 21.6867 110.115 20.8666 110.115 19.9262C110.115 18.9859 109.939 18.1668 109.59 17.4919ZM104.346 22.0143C103.973 21.5595 103.784 20.8573 103.784 19.926C103.784 18.9856 103.972 18.2812 104.344 17.831C104.705 17.3965 105.213 17.1844 105.897 17.1844C106.58 17.1844 107.087 17.3965 107.448 17.831C107.82 18.2812 108.009 18.9856 108.009 19.926C108.009 20.8562 107.82 21.5583 107.447 22.0143C107.087 22.4533 106.58 22.6676 105.897 22.6676C105.213 22.6676 104.706 22.4533 104.346 22.0143Z" fill="white"/>
+<path d="M63.7249 11.1308C63.7022 10.9805 63.5345 10.8204 63.3854 10.7854C62.8947 10.6726 62.5286 10.5925 62.0583 10.5498C61.5904 10.5047 61.1276 10.4822 60.6825 10.4822C59.7214 10.4822 58.8944 10.5874 58.2215 10.7927C57.5261 11.0032 56.9444 11.3289 56.4967 11.7598C56.0541 12.1831 55.7 12.7067 55.4395 13.3128C55.1891 13.8991 54.9968 14.5805 54.8678 15.3394L53.3882 24.1372C53.3705 24.2472 53.4009 24.3576 53.4742 24.4428C53.545 24.528 53.6512 24.5757 53.7625 24.5757H55.5759C55.7606 24.5757 55.9198 24.4428 55.9502 24.2624L56.575 20.5422H61.1906V24.1999C61.1906 24.4078 61.3601 24.5757 61.57 24.5757H63.3454C63.5553 24.5757 63.7248 24.4078 63.7248 24.1999V13.388C63.7249 13.0423 63.7249 11.1308 63.7249 11.1308ZM61.1908 13.3505V18.2529H56.9595L57.4298 15.4672C57.546 14.8286 57.7156 14.2975 57.9356 13.8967C58.138 13.5233 58.4338 13.2529 58.8361 13.0674C59.2535 12.872 59.8503 12.7743 60.609 12.7743C60.8315 12.7743 61.0264 12.7818 61.1956 12.7944C61.1929 12.9696 61.1908 13.1551 61.1908 13.3505Z" fill="white"/>
+</svg>
```

Comment:
Could we compress these images? for instance https://jakearchibald.github.io/svgomg/

<img width="397" alt="Screenshot 2021-10-01 at 11 38 34" src="https://user-images.githubusercontent.com/3165635/135599276-8f9d7407-aeb6-4a2b-9d82-4a7582b7ab6e.png">